packages/plugins/videos-ui/src/lib/shared/services/download/download-state.factory.ts

```
@@ -0,0 +1,22 @@
+				return new PendingState();
+			case DownloadStatus.DOWNLOADING:
+				return new DownloadingState();
+			case DownloadStatus.COMPLETED:
+				return new CompletedState();
+			case DownloadStatus.FAILED:
+				return new FailedState();
+			default:
+				throw new Error(`Unknown state for status: ${status}`);
+		}
+	}
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)